output-tests/README.md

```diff
@@ -2,15 +2,15 @@ These tests are intended to validate that implementations are correctly generati
 
 Output was initially specified with draft 2019-09.  It remained largely unchanged for draft 2020-12, but will receive an update with the next release.
 
-***NOTE** Although the formats didn't change between 2019-09 and 2020-12, the tests are replicated for 2020-12 because the `$schema` is different and implementations may (but shouldn't) produce different output.*
+***NOTE*** Although the formats didn't change much between 2019-09 and 2020-12, the tests are copied for 2020-12 because the `$schema` is different and implementations may (but shouldn't) produce different output.\*
```

Comment:
No, the formatting is correct.  Bold all-caps **NOTE** and the entire note in italics.

Reply:
If you prefer, you can change to 

```markdown
_**NOTE** blah blah blah_
```

(italics denoted with `_` instead)

Reply:
The formatting was not correct. If you look at the markdown version you will see that multiple paragraphs were bolded, because the wrong ending delimiter was used.

Reply:
I use this formatting throughout my personal markdown. I assure you, it is generally correct.  Use the underscores if you want.

Reply:
What underscores are you talking about? The problem that I corrected here was that the asterisks were mismatched, resulting in more content being bolded than was intended. You can see it in the github preview.

Reply:
They're not mismatched. Three at the beginning (one for italics, two for bold), two after `NOTE` (ending bold), and one at the end (ending italics).

My suggestion if it's not working for you is to replace the single asterisks with underscores.  So then you'd have an underscore at the beginning (for italics), two asterisks (for bold), two asterisks after `NOTE` (ending bold), and an underscore at the end (ending italics).

Reply:
It's not "not working for me", but more "not rendering properly on github".

Reply:
How is there room to argue about this sort of thing, it seems like the kind of thing that's easy to verify, or if not, just for each to post a screenshot of what you're talking about.

Here's what I see on current `main`:

<img width="992" alt="Screenshot 2022-12-30 at 13 55 15" src="https://user-images.githubusercontent.com/329822/210103579-0882d742-5a29-48ec-8fe6-96390ca39dfd.png">


That kind of formatting is "loud" to me (i.e. sticks out), which I noticed on the original PR but didn't care enough to mention, but seems to render fine (i.e. as intended, with bold NOTE and italic for the rest). So I don't see an issue personally. What are you referring to when saying it doesn't render properly? Can you show what you mean?

Reply:
("Sticks out" is what I was going for.  It's not a quote, so block quote formatting isn't right, but it still needs to be identified as separate from, yet pertaining to, the text.  It's a note, a clarification. There aren't a whole lot of options that markdown affords, so I came up with this.)